src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import AddTodo from './components/AddTodo';
```

Comment:
components name should be more meaningful and sensible.
```suggestion
import AddTaskField from './components/AddTaskField';
```